src/lib/Behat/PageElement/LeftMenu.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageElement;
+
+use PHPUnit\Framework\Assert;
+
+class LeftMenu extends Element
+{
+    private $buttonSelector = '.ez-sticky-container .btn';
+    private $menuSelector = 'ez-side-menu';
```

Comment:
Please move it to `$fields` array like in other Elements.
Also, is the lack of '.' in `$menuSelector` on purpose?

Reply:
1) 👍 
2) Nope, I've missed it - added together with a missing `verifyVisibility` call